ansible/roles/forc_api/files/backup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+NOW=$(date '+%y-%m-%d-%H%M')
+
+FILE=/etc/backup/backends-${NOW}.tar.gz
+tar czvf $FILE backends templates
```

Comment:
In /var/log/cron.log I get
> tar: backends: No such file or directory
> tar: templates: No such file or directory
> tar: error exit delayed from previous errors
when it runs automatically. Running the script manually works. Maybe try `/backends /templates` instead of `backends templates`?

Reply:
done